### PR TITLE
Fix/error handling

### PR DESCRIPTION
**Pull request by Deepak101094**
trying to fix error handling if  response failed..

### CODE REVIEW

File: src/components/MovieSearch.js [hunk 1]
```diff
@@ -25,74 +26,84 @@ function MovieSearch() {
...
           })
           .then((res) => {
-            setResources(res?.data);
-            if (res?.data?.Error) {
-              setError("Movie Not Found...!");
-              console.log(error);
+            setResources(res?.data ?? {});
+            if (res.status === 200) {
+              setSuccess(false);
+              setError("movie not found..");
```
Comment (shubuu97) [inline src/components/MovieSearch.js]: Capitalize Error Message

File: src/components/MovieSearch.js [hunk 2]
```diff
@@ -25,74 +26,84 @@ function MovieSearch() {
...
     }
 
-    setLoader(false);
+    setTimeout(() => {
+      setLoader(false);
+    }, 2000);
     setShowCard(true);
-    setSearchText("");
+
+    //setSearchText("");
```
Comment (shubuu97) [inline src/components/MovieSearch.js]: Remove comment

File: src/components/MovieSearch.js [hunk 3]
```diff
@@ -25,74 +26,84 @@ function MovieSearch() {
...
   };
 
   // when user click on enter button then this function will call..!
 
   const handleKeyDown = (event) => {
     if (event.keyCode === 13) {
       event.preventDefault();
```
Comment (shubuu97) [inline src/components/MovieSearch.js]: Move `event.preventDefault();` before `if`

File: src/components/Resources.js [hunk 4]
```diff
@@ -14,35 +16,49 @@ function Resources({ resources, error }) {
...
-              <small>Awards:{Awards} </small>
-            </p>
-            <p className="card-description"> {Plot} </p>
-          </div>
-        </div>
-      ) : (
-        <h2 style={{ color: "red", textAlign: "center" }}>{error}..</h2>
+  
+      <div className="card">
+      {/* <img
```
Comment (shubuu97) [inline src/components/Resources.js]: Remove commented code

**Review by shubuu97 — COMMENTED** [review]
Good Work, Please address these changes

File: src/components/MovieSearch.js (hunk 1 above)
Comment (Deepak101094) [reply src/components/MovieSearch.js]: done

File: src/components/MovieSearch.js (hunk 3 above)
Comment (Deepak101094) [reply src/components/MovieSearch.js]: done

File: src/components/Resources.js (hunk 4 above)
Comment (Deepak101094) [reply src/components/Resources.js]: done

File: src/components/MovieSearch.js (hunk 2 above)
Comment (Deepak101094) [reply src/components/MovieSearch.js]: done

**Review by shubuu97 — APPROVED** [review]

**Pull request merged**